ext/reflection/php_reflection.c

```diff
@@ -4336,7 +4336,7 @@ static int _adddynproperty(zval *ptr, int num_args, va_list args, zend_hash_key
 	zval *retval = va_arg(args, zval*);
 
 	/* under some circumstances, the properties hash table may contain numeric
-	 * properties (e.g. when casting from array). This is a WONT FIX bug, at
+	 * properties (e.g. when casting from array). This is a WON'T FIX bug, at
```

Comment:
That's debatable, as the bug tracker says “Wont fix” (and even WONTFIX isn't uncommon).

Reply:
I found some references of ` won't fix` in lower case only. Should we revert this one?

Reply:
I don't *really* mind. :)